### PR TITLE
Issue #1743 Fix wrong eval in docker-env

### LINKED ISSUE #1743: Incorrect eval $() in docker-env command when --no-proxy or --unset is used
This issue is nothing big. Flags --no-proxy and --unset works fine, but the presented eval $() command is wrong and thus in this context the instructions in the output of the command are incorrect. I have tried on bash and cmd, probably will be the same on all shells...

Current situation (similar for --no-proxy):
```
[agajdosi@dhcp-24-156 minishift]$ minishift docker-env --unset
unset DOCKER_TLS_VERIFY
unset DOCKER_HOST
unset DOCKER_CERT_PATH
unset DOCKER_API_VERSION
# Run this command to configure your shell:
# eval $(minishift docker-env)
```

Expected:
```
[agajdosi@dhcp-24-156 minishift]$ minishift docker-env --unset
unset DOCKER_TLS_VERIFY
unset DOCKER_HOST
unset DOCKER_CERT_PATH
unset DOCKER_API_VERSION
# Run this command to configure your shell:
# eval $(minishift docker-env --unset)
```

### PR DESCRIPTION
**Pull request by bhavin192**
fixes #1743

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
ok to test

**Comment by LalatenduMohanty** [comment]
The CI failed with below error . It is not related to this PR but this is not something I expected to fail. cc @praveenkumar 

```
:08:33   Scenario: Starting Minishift with anyuid and xpaas add-ons enabled # features/cmd-addons.feature:82
20:08:33     Given Minishift has state "Does Not Exist"                       # integration_test.go:164 -> *Minishift.shouldHaveState
20:13:19     When executing "minishift start" succeeds                        # integration_test.go:699 -> executingMinishiftCommandSucceedsOrFails
20:13:19     Command 'start' did not execute successfully. cmdExit: 1, cmdErr: Error during post cluster up configuration: Timed out to poll success state of persistent-volume-setup job
```

**Comment by budhrg** [comment]
> The CI failed with below error . It is not related to this PR but this is not something I expected to fail. cc @praveenkumar

Looks like 2 minutes is also not sufficient.

**Comment by budhrg** [comment]
retest this please

File: cmd/minishift/cmd/env.go [hunk 1]
```diff
@@ -114,19 +114,16 @@ func getConfigUnset(forceShell string, noProxy bool) (*DockerShellConfig, error)
...
 		shellCfg.NoProxyVar, shellCfg.NoProxyValue = shell.FindNoProxyFromEnv()
+		cmdLine = cmdLine + " --no-proxy"
 	}
 
-	prefix, suffix, delimiter := shell.GetPrefixSuffixDelimiterForUnSet(userShell)
-	shellCfg.Prefix = prefix
-	shellCfg.Suffix = suffix
-	shellCfg.Delimiter = delimiter
+	shellCfg.UsageHint = shell.GenerateUsageHint(userShell, cmdLine)
+	shellCfg.Prefix, shellCfg.Suffix, shellCfg.Delimiter = shell.GetPrefixSuffixDelimiterForUnSet(userShell)
```
Comment (praveenkumar) [inline cmd/minishift/cmd/env.go]: 👍

File: cmd/minishift/cmd/env.go [hunk 2]
```diff
@@ -88,7 +87,7 @@ func getConfigSet(api libmachine.API, forceShell string, noProxy bool) (*DockerS
 		}
 
 		noProxyVar, noProxyValue := shell.FindNoProxyFromEnv()
-
+		cmdLine = cmdLine + " --no-proxy"
```
Comment (praveenkumar) [inline cmd/minishift/cmd/env.go]: 👍

**Comment by LalatenduMohanty** [comment]
Thanks @bhavin192 .

**Pull request merged**